model_info/trwiki.goodfaith.md

```diff
@@ -1,81 +1,81 @@
 	Environment:
-	 - revscoring_version: '2.8.0'
-	 - platform: 'Linux-4.9.0-11-amd64-x86_64-with-debian-9.12'
+	 - revscoring_version: '2.8.2'
+	 - platform: 'Linux-4.19.0-0.bpo.14-amd64-x86_64-with-debian-9.4'
 	 - machine: 'x86_64'
-	 - version: '#1 SMP Debian 4.9.189-3+deb9u1 (2019-09-20)'
+	 - version: '#1 SMP Debian 4.19.171-2~deb9u1 (2021-02-08)'
 	 - system: 'Linux'
 	 - processor: ''
-	 - python_build: ('default', 'Sep 27 2018 17:25:39')
+	 - python_build: ('default', 'Apr  5 2021 09:00:41')
 	 - python_compiler: 'GCC 6.3.0 20170516'
 	 - python_branch: ''
 	 - python_implementation: 'CPython'
 	 - python_revision: ''
 	 - python_version: '3.5.3'
-	 - release: '4.9.0-11-amd64'
+	 - release: '4.19.0-0.bpo.14-amd64'
 	
 	Statistics:
-	counts (n=19319):
+	counts (n=22218):
 		label        n         ~True    ~False
 		-------  -----  ---  -------  --------
-		True     18499  -->    16848      1651
-		False      820  -->      292       528
+		True     21035  -->    18434      2601
+		False     1183  -->      373       810
 	rates:
 		              True    False
 		----------  ------  -------
-		sample       0.958    0.042
+		sample       0.947    0.053
 		population   0.954    0.046
-	match_rate (micro=0.85, macro=0.5):
+	match_rate (micro=0.818, macro=0.5):
 		  True    False
 		------  -------
-		 0.885    0.115
-	filter_rate (micro=0.15, macro=0.5):
+		  0.85     0.15
+	filter_rate (micro=0.182, macro=0.5):
 		  True    False
 		------  -------
-		 0.115    0.885
-	recall (micro=0.898, macro=0.777):
+		  0.15     0.85
+	recall (micro=0.868, macro=0.781):
 		  True    False
 		------  -------
-		 0.911    0.644
-	!recall (micro=0.656, macro=0.777):
+		 0.876    0.685
+	!recall (micro=0.694, macro=0.781):
 		  True    False
 		------  -------
-		 0.644    0.911
-	precision (micro=0.948, macro=0.62):
+		 0.685    0.876
+	precision (micro=0.947, macro=0.597):
 		  True    False
 		------  -------
-		 0.981    0.259
-	!precision (micro=0.292, macro=0.62):
+		 0.983    0.211
+	!precision (micro=0.247, macro=0.597):
 		  True    False
 		------  -------
-		 0.259    0.981
-	f1 (micro=0.918, macro=0.657):
+		 0.211    0.983
+	f1 (micro=0.899, macro=0.625):
 		  True    False
 		------  -------
-		 0.945    0.369
-	!f1 (micro=0.396, macro=0.657):
+		 0.927    0.323
+	!f1 (micro=0.351, macro=0.625):
 		  True    False
 		------  -------
-		 0.369    0.945
-	accuracy (micro=0.898, macro=0.898):
+		 0.323    0.927
+	accuracy (micro=0.868, macro=0.868):
 		  True    False
 		------  -------
-		 0.898    0.898
-	fpr (micro=0.344, macro=0.223):
+		 0.868    0.868
+	fpr (micro=0.306, macro=0.219):
 		  True    False
 		------  -------
-		 0.356    0.089
-	roc_auc (micro=0.911, macro=0.911):
+		 0.315    0.124
+	roc_auc (micro=0.893, macro=0.893):
 		  True    False
 		------  -------
-		 0.911    0.911
-	pr_auc (micro=0.964, macro=0.653):
+		 0.893    0.894
+	pr_auc (micro=0.961, macro=0.637):
```

Comment:
Minor loss in fitness could be because of the data merging work I described above.  It might get cleaned up when the rest of the sample is merged in.